starwars/src/components/CharacterInfo.js

```diff
@@ -6,10 +6,16 @@ const CharacterInfo = ({ data }) => (
     {data.birth_year === 'unknown' ? '' : <p>Born in {data.birth_year}</p>}
```

Comment:
- Great job on the sprint challenge Jakub. You successfully completed the MVP by fetching data from 3rd party API and displayed it on the webpage. You have properly split functionality into smaller components and passed necessary data as props.
- Your project is styled properly and design looks good. 
- You have also gone above and beyond and fetch the Api links from incoming data. You are able to load and display films for every character.Great job on that! 
Your code is clean and Your git workflow is fluid.
Overall:
  * Able to fetch and display data (MVP)
  * Good answers
  * Clean and modular code with multiple components
  * Implemented styling
  * Responsive design
  * Fluid git workflow